webapp/src/Entity/Contest.php

```diff
@@ -389,9 +389,9 @@ public function getExternalid(): ?string
         return $this->externalid;
     }
 
-    public function setName(string $name): Contest
+    public function setName(?string $name): Contest
     {
-        $this->name = $name;
+        $this->name = strval($name);
```

Comment:
I think I'd prefer `(string)$name` or even `$name ?? ''`

Reply:
But maybe even better is the thing in the form types I suggested on Slack, with `empty_data`

Reply:
> But maybe even better is the thing in the form types I suggested on Slack, with `empty_data`

The reason I didn't went for that is that we now handle the case deeper in the code so if there is another way of passing the wrong data we still handle that in a reasonable way.

> I think I'd prefer (string)$name or even $name ?? ''

Fine with the first one,

Reply:
@nickygerritsen I've now also done what you mentioned in slack. I've left the work on the `set*Name` functions as I think the extra protection is useful.

Reply:
I’m not really sure if I agree with passing wrong data (we also don’t have stuff like passing strings to functions that accept int nor should we) but I’m also not strongly against it